huggan/model_card_template.md

```diff
@@ -0,0 +1,50 @@
+---
+tags:
+- huggan
+- gan
+task: image-generation
```

Comment:
WDYT...should we link to https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts somewhere around here? For example, if the idea is to generate an image based on text, we need to use text-to-image. But for example, if this is image-to-image, we will need to add that to `Types.ts`. Since widgets will not work out of the box, I don't think we need to have all of them in `Types.ts`. During the sprint we can see which are the patterns and start adding tags based on that

Reply:
Sure! I added a PR [here](https://github.com/huggingface/hub-docs/pull/80) for the two tasks `image-generation` and `image-to-image` for now. We'll add this link later once that gets merged.